website/docs/concepts/function-components/properties.mdx

```diff
@@ -263,6 +263,28 @@ fn App() -> Html {
 }
 ```
 
+## Automatically generate properties (autoprops)
+
+In order to streamline your development process, you can also use the macro
+`#[autoprops]` that will automatically generate the `Properties` struct for you.
+
+```rust
+use yew::prelude::*;
+
+#[autoprops]
+#[function_component]
+fn Greetings(
+    #[prop_or("Hello".into())]
+    message: AttrValue,
+    #[prop_or("World".into())]
+    name: AttrValue,
```

Comment:
```suggestion
    #[prop_or("Hello".into())]
    message: &AttrValue,
    #[prop_or("World".into())]
    name: &AttrValue,
```

It might actually be a bit better to use references for AttrValue because you can reuse the variable multiple times directly without moving it. Types that are Copy should always use the cloning approach though.